sc2/client.py

```diff
@@ -336,6 +336,19 @@ def in_game(self):
             return [[AbilityId(a.ability_id) for a in b.abilities] for b in result.query.abilities][0]
         return [[AbilityId(a.ability_id) for a in b.abilities] for b in result.query.abilities]
 
+    async def query_available_abilities_with_tag(
+            self, units: Union[List[Unit], Units], ignore_resource_requirements: bool = False
+    ) -> Dict[Set[AbilityId]]:
```

Comment:
The return type hint is wrong and should be: `Dict[int, Set[AbilityId]]`

Reply:
Fixed